packages/client/src/utils/generateInFolder.ts

```diff
@@ -24,19 +24,13 @@ const del = promisify(rimraf)
 
 export interface GenerateInFolderOptions {
   projectDir: string
-  useLocalRuntime?: boolean
-  transpile?: boolean
```

Comment:
What was this used for?

Reply:
`useLocalRuntime` is dead code (always set to false) and was removed https://github.com/prisma/prisma/pull/22630/files#diff-48c06b9c0b442da6779812b9b0acdeaff470e209e1fd3ff31ba4037061c4cb8dL85

`transpile` is always set to true, which also generates the client as it does everywhere else, so it is not actually a useful setting nor needed. Setting `transpile` to `false` prevents the engine copy within the tests, and we never set it to false, also because we don't need this https://github.com/prisma/prisma/pull/22630/files#diff-3c7fec7e990cab78b875f4fcce9979640a5fe3004db9dd082578dac14874d5deL296

Both of these settings are testing-related only.